ddp-workspace/projects/ddp-sdk/src/lib/components/user/activities/userActivities.component.ts

```diff
@@ -169,8 +169,8 @@ export class UserActivitiesComponent implements OnInit, OnDestroy, OnChanges, Af
   @Input() displayedColumns: Array<DashboardColumns> = ['name', 'summary', 'date', 'status', 'actions'];
   @Output() open: EventEmitter<string> = new EventEmitter();
   public dataSource: UserActivitiesDataSource;
-  public loaded: boolean;
-  private states: Array<ActivityInstanceState> | null;
+  public loaded = false;
+  private states: Array<ActivityInstanceState> | null = null;
   private studyGuidObservable: BehaviorSubject<string | null>;
   private loadingAnchor: Subscription;
 
```

Comment:
Unrelated to mailing address, but noticed while working on mailing address and going back and forth from dashboard on testboston that occassionally we would get errors where `states` was being used while still undefined. Providing a couple of initial values solved problem.